uwblueprint/planet-read
# Convert story translation page to story page

## Notion ticket link



[Convert story translation page to story page](https://www.notion.so/uwblueprintexecs/Convert-story-translation-page-to-story-page-14548e9df43845298bc7286354780f28)



## Implementation description

- repurposed `ManageStoryTranslationPage` into `ManageStoryPage`
- modified `storyTranslations` query to accept filtering by `storyId`
- modified `StoryTranslationsTable` to work on the `ManageStoryPage`
- page route changed to `/story/:storyId`

**Note**: currently we don't have delete story functionality so the button doesn't do anything for now, added a TODO comment for it



## Steps to test

1. Login as admin
2. Go to `/story/1` or any other story
3. Verify that the page layout looks good and story translations appear
4. Try editing the story fields and saving changes, verify that it works as normal
5. Verify that deleting a story translation works properly



## What should reviewers focus on?
- does the styling look okay?


## Checklist
